crates/wast-util/src/lib.rs

```diff
@@ -509,12 +507,10 @@ impl WastTest {
                 "spec_testsuite/simd_f32x4_arith.wast",
                 "spec_testsuite/simd_f32x4_cmp.wast",
                 "spec_testsuite/simd_f32x4_pmin_pmax.wast",
-                "spec_testsuite/simd_f32x4_rounding.wast",
                 "spec_testsuite/simd_f64x2.wast",
                 "spec_testsuite/simd_f64x2_arith.wast",
                 "spec_testsuite/simd_f64x2_cmp.wast",
                 "spec_testsuite/simd_f64x2_pmin_pmax.wast",
-                "spec_testsuite/simd_f64x2_rounding.wast",
```

Comment:
Oh this'll want to stay as these are test exemptions for a different compiler, Winch, unrelated to Pulley. Only the ones above are the ones affecting Pulley.

Also it looks like a merge conflict has creeped in due to this file, so if you want to rebase that should help resolve the conflict.

Reply:
resolved it.